tests/Colopl/Spanner/TransactionTest.php

```diff
@@ -260,42 +252,34 @@ public function testLockTimeout(): void
         $caughtException = null;
         try {
             $conn->transaction(function () use ($conn2, $qb, $qb2, $mutation) {
-                // SELECTing within a read-write transaction causes row to aquire shared lock
-                $findRow = $qb->where('userId', $mutation['userId'])->first();
+                // SELECTing within a read-write transaction causes row to acquire shared lock
+                 $qb->where('userId', $mutation['userId'])->first();
 
                 $conn2->transaction(function () use ($qb2, $mutation) {
-                    // will timeout and result in AbortedException since row is locked
+                    // This will time out and result in AbortedException since row is locked
                     $qb2->where('userId', $mutation['userId'])->update(['name' => $mutation['name']]);
                 }, 1);
             }, 1);
-        } catch (\Exception $ex) {
+        } catch (Exception $ex) {
             $caughtException = $ex;
         }
 
         $this->assertInstanceOf(AbortedException::class, $caughtException);
-        $this->assertStringContainsString('ABORTED', $caughtException?->getMessage() ?? '');
+        $this->assertStringContainsString('ABORTED', $caughtException->getMessage());
     }
 
     public function testAbortedException(): void
     {
-        $committedCount = 0;
-        $this->app['events']->listen(TransactionCommitted::class, function ($ev) use(&$committedCount) {
-            $committedCount++;
-        });
-
-        $rolledBackCount = 0;
-        $this->app['events']->listen(TransactionRolledBack::class, function ($ev) use (&$rolledBackCount) {
-            $rolledBackCount++;
-        });
+        Event::fake();
 
         $conn = $this->getDefaultConnection();
         try {
-            $conn->transaction(function () {
-                throw new AbortedException('abort');
-            });
-        } catch (AbortedException $ex) {}
+            $conn->transaction(fn() => throw new AbortedException('abort'));
+        } catch (AbortedException) {
+
+        }
 
-        $this->assertEquals(0, $committedCount);
-        $this->assertEquals(10, $rolledBackCount);
+        Event::assertDispatchedTimes(TransactionCommitted::class, 0);
+        Event::assertDispatchedTimes(TransactionRolledBack::class, 10);
```

Comment:
```suggestion
        Event::assertDispatchedTimes(TransactionRolledBack::class, Database::MAX_RETRIES));
```

`use Google\Cloud\Spanner\Database` is also needed.